rfcs/text/0052/gen_ai.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- name: gen_ai
+  fields:
+    - name: system_instructions
+      type: flattened
+      description: The system message or instructions provided to the GenAI model separately from the chat history.
+      example: TODO
+      level: extended
+      beta: This field reuse is beta and subject to change.
```

Comment:
why it’s field `reuse`?

Reply:
Sorry, I was incorrect before, the text should be `This field is beta and subject to change.` for all the beta fields here